eng/targets/Settings.props

```diff
@@ -194,4 +194,8 @@
       </PropertyGroup>
     </When>
   </Choose>
+
+  <PropertyGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
+    <DefineConstants>DotNetBuildFromSource</DefineConstants>
```

Comment:
Consider appending to `DefineConstants` instead of replacing it.

Reply:
Think this is a requirement not a consideration. It will overwrite the use of `$(DefineConstants)` above. 

---
In reply to: [319689693](https://github.com/dotnet/roslyn/pull/38425#discussion_r319689693) [](ancestors = 319689693)

Reply:
In general define constants should be all upper case

---
In reply to: [319691497](https://github.com/dotnet/roslyn/pull/38425#discussion_r319691497) [](ancestors = 319691497,319689693)